packages/dual-channel/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@twreporter/dual-channel",
+  "version": "2.0.1",
+  "description": "Build Dual Channel style infographic web page on demand",
+  "scripts": {
+    "build": "make build",
+    "clean": "make clean",
+    "dev": "make dev",
+    "dev-server": "make dev-server"
+  },
+  "license": "MIT",
+  "main": "lib/index.js",
+  "devDependencies": {
```

Comment:
Note: After discussion with @taylrj and @nickhsine, we decided to use shared root babel and webpack config and dependencies for each subpackage by now. And the `devDependecies` in each subpackage will only includes the ones that not are not contained in the root `package.json`.

Reply:
@YuCJ @taylrj 
Please see this commit https://github.com/twreporter/orangutan/pull/6/commits/51b9880f5fe3911895a3cf7e5abd94fb4fea82ef .
I move devDependencies, which are used by other sub-packages,  from sub-package level to root level.
We can discuss whether some of them should stay at sub-pacakge level or not.